apps/oxlint/src/output_formatter/github.rs

```diff
@@ -79,3 +81,33 @@ fn escape_property(value: &str) -> String {
     }
     result
 }
+
+#[cfg(test)]
+mod test {
+    use oxc_diagnostics::{reporter::DiagnosticReporter, NamedSource, OxcDiagnostic};
+    use oxc_span::Span;
+
+    use super::GithubReporter;
+
+    #[test]
+    fn reporter_finish() {
+        let mut reporter = GithubReporter;
+
+        let result = reporter.finish();
+
+        assert!(result.is_none());
+    }
+
+    #[test]
+    fn reporter_error() {
+        let mut reporter = GithubReporter;
+        let error = OxcDiagnostic::warn("error message")
+            .with_label(Span::new(0, 8))
+            .with_source_code(NamedSource::new("file://test.ts", "debugger;"));
+
+        let result = reporter.render_error(error);
+
+        assert!(result.is_some());
+        assert_eq!(result.unwrap(), "::warning file=file%3A//test.ts,line=1,endLine=1,col=1,endColumn=1,title=oxlint::error message\n");
```

Comment:
`endColumn=1` is wrong, `oxc_diagnostics::reporter:Info` does not contain and end column